solvebio/client.py

```diff
@@ -107,12 +107,23 @@ def __init__(self, host=None, token=None, token_type='Token',
         # intermittent connection errors.
         retries = Retry(
             total=5,
+            connect=5,
```

Comment:
Does this have an impact if we already set `total`?

Can we increase the backoff factor to `2`?